roles/setup_hammerdb/tasks/configure_postgres_params.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: Update PostgreSQL parameters for TPROC-C
+  include_role:
+    name: manage_dbserver
+    tasks_from: manage_postgres_params
+  vars:
+    pg_postgres_conf_params:
+    - name: max_connections
+      value: 300
+    - name: shared_buffers
+      value: "16384MB"
+    - name: work_mem
+      value: "64MB"
+    - name: maintenance_work_mem
+      value: "620MB"
+    - name: effective_cache_size
+      value: "45 GB"
+    - name: random_page_cost
+      value: 1.25
+    - name: effective_io_concurrency
+      value: 200
+    - name: checkpoint_timeout
+      value: "15min"
+    - name: checkpoint_completion_target
+      value: 0.9
+    - name: max_wal_size
+      value: "30GB"
+    - name: min_wal_size
+      value: "10GB"
+    - name: wal_buffers
+      value: "512MB"
+    - name: jit
+      value: on
```

Comment:
I think we should move this part to `roles/setup_hammerdb/defaults/main.yml`

Reply:
Hi @mw2q, Can you please check if the autotuning role can replace the above setting requirements?

Reply:
Also can you please use FQCN based module call. For example, `template` can be called as `ansible.builtin.template`